app/views/exam_templates/_form.html.erb

```diff
@@ -0,0 +1,21 @@
+<%= form_for exam_template,
+             as: :exam_template,
+             url: assignment_exam_templates_path,
+             html: {multipart: true} do |f| %>
+    <%= f.error_messages%>
+    <div class='template-pane'>
+      <h4>
+        <%= link_to t('exam_templates.download'), {:action=>'download', :controller=>'exam_templates', :id=>exam_template.id} %>
```

Comment:
Use a path helper. See http://guides.rubyonrails.org/routing.html#adding-more-restful-actions for what is generated when you create a new route (like `download`). You are looking for a method that ends with `_path`.